### PR TITLE
Add cnx robots.txt to /var/www/webview

### PR DESCRIPTION
**Pull request by karenc**
We had robots.txt in cnx-archive which was not really the right place to
put it.  DevOps prefer to have robots.txt in cnx-deploy so they can
configure it.

### CODE REVIEW

File: roles/robots_txt/vars/main.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    - /*/pdf$
+    - /*/epub$
+    - /*/complete$
+    - /*/offline$
+    - /*?format=*$
+    - /*/multimedia$
+    - /*/lens_add?*$
+    - /lens_add
+    - /*/lens_view/*$
+    - /content/*view_mode=statistics$
```
Comment (pumazi) [inline roles/robots_txt/vars/main.yml]: I'm not a fan of buried configuration. But maybe I'm looking at this stuff wrong.

Comment (karenc) [reply roles/robots_txt/vars/main.yml]: You mean you would prefer to move this to group vars?  I can also move this to the template (but move the user agent config to group vars)

**Comment by pumazi** [comment]
We should get this into place soon'ish since the current robots.txt points to `/sitemap.xml`, which is an invalid url. I'd create a separate issue, but I don't want to muddy things more than they might already be (that is, multiple input steams to the devops team). So let's just try to get this pushed through.

File: roles/robots_txt/vars/main.yml (hunk 1 above)
Comment (reedstrm) [reply roles/robots_txt/vars/main.yml]: Hmm, there's actually two levels of config here - user-agents to disallow would be universal, but which paths are not-of-interest for crawling is an aspect of the web-apps deployed, so shouldn't those apps configure fragments for robots.txt, like the haproxy config does?

Comment (reedstrm) [reply roles/robots_txt/vars/main.yml]: So, I think I'm voting "vars" right?

Comment (karenc) [reply roles/robots_txt/vars/main.yml]: These are all legacy urls, right?

Comment (reedstrm) [reply roles/robots_txt/vars/main.yml]: yes

**Pull request merged**